roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,10 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        multicastTTL = topology->getPropInt("@multicastTTL", 1);
+        if (roxieMulticastEnabled && traceLevel && multicastTTL > 1)
```

Comment:
I'd be inclined to trace whatever the value of multicastTTL is, Also, should you check for/reject invalid values?